### PR TITLE
Fix dconf_gnome_disable_geolocation script and add missing dconf remedation scripts

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Fix dconf_gnome_disable_geolocation script and add missing dconf remediation scripts

#### Rationale:

- Script was erroring out on remediation and a couple of remediation scripts were missing

### CODE REVIEW

File: shared/fixes/bash/dconf_gnome_disable_wifi_create.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# platform = Red Hat Enterprise Linux 7
```
Comment (mpreisler) [inline shared/fixes/bash/dconf_gnome_disable_wifi_create.sh]: if it's rhel7 only don't put it in shared, if it's universal don't make it rhel7 only

Comment (redhatrises) [reply shared/fixes/bash/dconf_gnome_disable_wifi_create.sh]: The majority of rules in shared are rhel7 only.

**Comment by redhatrises** [comment]
@yuumasato if still possible need this in 0.1.39

File: shared/fixes/bash/dconf_gnome_disable_wifi_create.sh (hunk 1 above)
Comment (redhatrises) [reply shared/fixes/bash/dconf_gnome_disable_wifi_create.sh]: @mpreisler added to Fedora as well.

**Comment by yuumasato** [comment]
@redhatrises sure, just this one.

**Comment by redhatrises** [comment]
yup @yuumasato just this one.

**Comment by yuumasato** [comment]
Looks good, Ack.

File: shared/fixes/bash/dconf_gnome_disable_wifi_create.sh (hunk 1 above)
Comment (mpreisler) [reply shared/fixes/bash/dconf_gnome_disable_wifi_create.sh]: > The majority of rules in shared are rhel7 only.

yes and that sucks, let's not make it even worse :D

**Pull request merged**